pages/community.tsx

```diff
@@ -0,0 +1,242 @@
+          {kinds.map((key) => (
+            <ResourceLink
+              key={key}
+              kind={key}
+            />
+          ))}
+        </div>
+      </StandardPageWidth>
+      {/* events */}
+      <ColorModeProvider mode="light">
+        <div
+          className={classNames(
+            'bg-fill-zero',
+            'p-large py-xxxlarge md:p-xxxxxlarge md:pt-xxxxxlarge'
+          )}
+        >
+          <StandardPageWidth>
+            <ResponsiveText
+              className=" mb-xxlarge text-center"
+              as="h2"
+              textStyles={{
+                '': 'mTitle2',
+                sm: 'mHero2',
+              }}
+            >
+              Plural Events
+            </ResponsiveText>
+            {events.length ? (
+              <EventsGrid events={events} />
+            ) : (
+              <Card
+                fillLevel={1}
+                className={classNames(
+                  'flex',
+                  'justify-center p-large md:p-xxxxlarge'
+                )}
+              >
+                <ResponsiveText
+                  as="span"
+                  textStyles={{ md: 'mSubtitle1', '': 'mBody2Bold' }}
+                  className="m-large md:m-xxxxlarge"
+                >
+                  No planned events at this time
+                </ResponsiveText>
+              </Card>
+            )}
+          </StandardPageWidth>
+        </div>
+      </ColorModeProvider>
+    </>
+  )
+}
+
+const resources = {
+  gitHub: {
+    url: 'https://github.com/pluralsh/plural',
+    title: 'GitHub',
+    icon: <GitHubLogoIcon />,
+  },
+  youtube: {
+    url: 'https://www.youtube.com/channel/UCt8xwvJH4f4c6g8J1Y6ZmJQ',
+    title: 'YouTube',
+    icon: <YouTubeIcon />,
+  },
+  docs: {
+    url: 'https://docs.plural.sh',
+    title: 'Docs',
+    icon: <DocumentIcon />,
+  },
+} as const
+
+const kinds = Object.keys(resources) as Array<keyof typeof resources>
+
+function ResourceLink({ kind }: { kind: keyof typeof resources }) {
+  const theme = useTheme()
+
+  return (
+    <Button
+      secondary
+      as="a"
+      className="flex grow items-center justify-start gap-medium"
+      href={resources[kind].url}
+      target="_blank"
+      rel="noopener noreferrer"
+      style={{ justifyContent: 'flex-start', padding: theme.spacing.large }}
+    >
+      <IconFrame
+        type="floating"
+        style={{
+          display: 'inline-flex',
+          marginRight: theme.spacing.xsmall,
+        }}
+        icon={resources[kind].icon}
+        size="medium"
+      />
+      <span>{resources[kind].title}</span>
+    </Button>
+  )
+}
+
+function EventsGrid({ events }: { events: PluralEvent[] }) {
+  const locale = useRouter().locale || 'en-us'
+
+  return (
+    <div
+      className="grid grid-cols-1 gap-xxlarge md:grid-cols-2 xl:grid-cols-3"
+      style={{ gridAutoRows: '1fr' }}
+    >
+      {events.map((event) => (
+        <a
+          key={event.id}
+          href="#"
```

Comment:
should be the event url right? or is there not a page for it

Reply:
we wont have any events for now and the events don't have url configured in directus yet. I will change it when we start adding events